Tests/Package.swift

```diff
@@ -0,0 +1,15 @@
+// swift-tools-version:5.5
+//
+//  Package.swift
+//
+//
+//  Created by Andrés Boedo on 5/7/24.
+//
+// This empty file prevents Xcode from picking up the files in the Tests library
+// when importing through SPM. This is helpful so that users of the SDK
+// don't inadvertently get StoreKit Configuration files added to the project, which
+// can lead to confusion.
+
+import PackageDescription
+
+let package = Package()
```

Comment:
just wondering but what is this file for?

Reply:
oh just saw in the description:

> Adds a new Package.swift file at the root of the Tests folder, which in turn tells SPM to treat that folder as a separate package, and thus the files in there don't get picked up automatically when importing RevenueCat.

Reply:
haha yeah, exactly. Kinda reminds me of the `Dummy.swift` file we have in some of our repos (although it's for a different purpose of course)